### DISCUSSION TITLE
Limit of box quantity

### DISCUSSION THREAD

**Discussion by faradey** [dvdoug/BoxPacker#226] [Ideas]
Hello,
sometimes there is a need to limit the number of boxes due to the fact that there is only a certain quantity in the warehouse of each type of boxes. Now we cannot regulate this in any way.
For example, there are two types of boxes
1.with dimensions 12x12x12 in the amount of 1 piece
2.with dimensions 20x15x15 in the amount of 2 pieces

if there are a lot of goods and they have dimensions of 5x5x5, then several boxes of the first type will be selected. But there is only one such box in the warehouse.
How can this problem be solved? If now there is no such functionality, then can we hope for the appearance of such functionality in the future?

Thank you

**Comment by dvdoug** [comment]
Hello

If I understand your problem correctly, https://www.boxpacker.io/en/stable/advanced-usage.html#limited-supply-boxes should do what you need?

**Reply by faradey** [reply]
Oh, apparently I missed it in the manual. Yes, it is worked. Thank you.
